tests/db_tests/test_aws_access.py

```diff
@@ -28,7 +20,7 @@ def test_list_buckets():
 
 @pytest.mark.skip(reason="Test should only be run locally in dev environment")
```

Comment:
is this an actual local test we want to run as standard? Or just some debugging tool? If the former we could not skip it, but check the env and only run if local dev? Otherwise just pass. Is that feasible?

Reply:
More of a debugging tool I think. Otherwise we will have to have the localstack container running every time we run the tests locally in order to pass